processors/shop_hodinkee_processor.rb

```diff
@@ -3,32 +3,54 @@
 require_relative '../models/shop_hodinkee_model'
 
 class ShopHodinkeeProcessor < BaseProcessor
+  attr_accessor :content, :item_content, :page
+
   HOST = 'https://shop.hodinkee.com'
   PAGINATION_SELECTOR = '/collections/watches?page='
```

Comment:
please, add `/` slash to the end to `Host` and remove from selector value from beggining